### DISCUSSION TITLE
Supporting parameters for calculation of pKa- and pH dependent solubilitiy scale factor or logD factor

### DISCUSSION THREAD

**Discussion by nina-nauwelaerts** [Open-Systems-Pharmacology/Forum#1816] [Questions & Problems]
I'm looking into the supporting parameters (F1-F3 and K1-K8) for calculation of pKa- and pH- dependent solubilty scale factor or logD factor. Is there any background reading about these parameters and/or could anyone help me understand how they are used in OSP?

**Comment by wschm** [comment]
Unfortunately there is no literature source, I know of, which could explain this. Let me try to explain it in my words. I hope it is not to complicated.

The charging status at a given pH for molecules with multiple chargeable moieties is calculated by calculating the for each possible number of charges its probability. For example, for a molecule containing three chargeable moieties, two acidic and one basic, depending on the pH four different charging stages (i.e. macrospecies) -1, 0, 1 and 2 are possible. Each of the three macrospecies can, however, be composed of different microspecies. E.g. the charge 0 can be achieved by three cases: 1. All moieties are uncharged, 2. The basic moiety and one or the other of the two acidic ones are fully charged. Thus, there are 2^3 = 8 possible microspecies for which the charging status at the given pH has to be calculated and the fractions with which these microspecies occur at the pH is calculated as the product of the probabilities for each moiety to be charged. The latter being determined based on the pKa for each moiety using the Henderson-Hasselbach equation. These eight fractions are the parameters K1 – K8. Finally, for each macrospecies its fraction (= parameters F1 – F3) of occurrence is given by the sum of the fractions for microspecies, by which it can be realized. Please note that for the calculation of logD and solubility it is assumed that each charge, independent of its sign, changes the phys-chem properties by the same factor. For this reason only three “microspecies” are considered instead of four, by not differentiating between +1 and -1.
